<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirstWindow</class>
 <widget class="QDialog" name="FirstWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>bikes</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="4">
    <widget class="QPushButton" name="pushButton_5">
     <property name="text">
      <string>Поиск</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="lineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="3">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="6" column="2" colspan="3">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QPushButton" name="pushButton_6">
     <property name="text">
      <string>Объединить товары</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="pushButton_4">
     <property name="text">
      <string>Перейти к товару</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
